virt/virt-2-4-release-notes.adoc

```diff
@@ -107,6 +112,9 @@ from being inadvertently started before an upload has completed.
 //CNV-5371 - Import existing virtual machines from Red Hat Virtualization to OpenShift Virtualization.
 * You can import a single Red Hat Virtualization virtual machine by using the virtual machine wizard or the CLI.
 
+//CNV-5463 - New API Subgroup for Virt
+* Every component in {VirtProductName} now uses its own API sub-group, `<component_name>.kubevirt.io`.
```

Comment:
I think it might actually just be "subgroup" without the hyphen. There's a module with "subgroup" in the title and I don't see any instances of "sub-group"

Reply:
Done.